troubadix/plugins/http_links_in_tags.py

```diff
@@ -197,5 +197,12 @@ def check_to_continue(http_link_match_group: AnyStr) -> bool:
             return True
         if "https://www.phishingtarget.com@evil.com" in http_link_match_group:
             return True
+        # e.g.:
+        # Since gedit supports opening files via 'http://' URLs (through GNOME vfs)
```

Comment:
```suggestion
        # Since gedit supports opening files via 'http://' URLs
        # (through GNOME vfs)
```

Reply:
This should be enough and avoiding to have two lines:

```suggestion
        # Since gedit supports opening files via 'http://' URLs
```